src/lib/types.ts

```diff
@@ -73,3 +73,7 @@ export interface MarkerLayerEventInfo extends MarkerClickInfo {
 
 export type DeckGlAccessor<DATA, RETVAL> = RETVAL | ((data: DATA) => RETVAL);
 export type DeckGlColorAccessor<DATA> = DeckGlAccessor<DATA, number[]>;
+
+export type Scheme = 'xyz' | 'tms';
```

Comment:
Are these types exported by MapLibre itself? If so I think it’s better to use those so that we are guaranteed to stay in sync with any new updates. 

If not, then this is great.

Reply:
Scheme is not as it is defined as a string in MapLibre, but `DEMEncoding` is and has now been replaced in the last commit.